app/experimenter/experiments/tests/api/v3/test_views.py

```diff
@@ -12,6 +12,24 @@
 from experimenter.bugzilla.tests.mixins import MockBugzillaTasksMixin
 
 
+FAKE_VARIANTS = [
```

Comment:
If this is only used in one test case you can just stick this right on the test class, either as a class var like

```py
class TestWhatever
    FAKE_VARIANTS = { .... }
```

or

```
class TestWhatever
    def setUp(self):
        self.variants_data = { .... }
```

and then you can refer to it in the test as `self.FAKE_VARIANTS` or `self.variants_data`